students/stuartkershaw/lesson03/mailroom.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+donors = [
+    'Anne Hathaway', 'Geronimo Jackson', 'Dingo Dogson',
+    'Leo Kottke', 'Flask McCreole', 'Piglet Norquist',
+    [[4000, 6250], [150, 400, 1250], [150, 300, 100],
+     [75, 125], [1000], [20, 15, 30]]
+]
+
+
+def compose_email(donor_list, index):
+    i = index
+    dl = donor_list
+    message = f"Dear {dl[i]}, thanks so much \
+for your generous donation in the amount of: ${dl[-1][i][-1]}."
+    print(message)
+
+
+def get_selection():
+    options = 'Please select a, b, or c:\n\
+a) send a thank you\nb) create a report\nc) quit\n'
+    while True:
+        try:
+            selection = input(options)
+            while selection not in ['a', 'b', 'c']:
+                selection = input(options)
+            return selection
+        except ValueError:
```

Comment:
I may be missing something but I don't see where a `ValueError` would be raised in the `try` block here. Seems like any 'bad' input would be filtered out in your `while` loop check